DEVELOPMENT_GUIDE.md

```diff
@@ -21,6 +21,41 @@ This will start a new Gitpod workspace, and immediately kick off a build of the
 
 Gitpod is free for 50 hours per month - make sure to stop your workspace when you're done (you can always resume it later, and it won't need to run the build again).
 
+5-Minute Tutorial
+-----------------
+
+This tiny tutorial will show you how to make a small change to SAM CLI.
+
+See the rest of this document for a more thorough introduction.
+
+1. Clone SAM CLI:
+
+    ```bash
+    git clone git@github.com:aws/aws-sam-cli.git
```

Comment:
Should we do git clone using `https` instead of `ssh`?

Reply:
It's using the Git protocol, which IIRC authenticates properly if the credentials are available. It also work without any write permissions.

Reply:
i prefer ssh instead of https because of convenience of not putting passwords everytime.

Reply:
Interesting... I don't need to put password for https and it works out of box. But for ssh, I need to set up a key. Not sure if it's related to my setup. As long as it's new hire friendly, I'm good either way.